lib/flex_commerce_api/json_api_client_extension/remote_builder.rb

```diff
@@ -1,10 +1,10 @@
 module FlexCommerceApi
   module JsonApiClientExtension
     class RemoteBuilder < ::JsonApiClient::Query::Builder
-      def initialize(klass, path: klass.path, connection: klass.connection)
-        super(klass)
-        self.connection = connection
-        self.path = path
+      def initialize(klass, opts = {})
```

Comment:
Some changes to bring us in line with interface changes within the `json_api_client` gem we're extending